File: src/main/java/com/team3/gudit/goods/constant/PathConstant.java
package com.team3.gudit.goods.constant;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

import java.nio.file.Path;
import java.nio.file.Paths;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class PathConstant {
    public static final Path THUMBNAIL_DIRECTORY = Paths.get("thumbnails").toAbsolutePath().normalize();
}

File: src/main/kotlin/com/team3/gudit/goods/constant/PathConstant.kt
package com.team3.gudit.goods.constant

import java.nio.file.Path
import java.nio.file.Paths

class PathConstant private constructor() {
    companion object {
        @JvmField
        val THUMBNAIL_DIRECTORY: Path =
            Paths.get("thumbnails")
                .toAbsolutePath()
                .normalize()
    }
}
